src/main/java/de/retest/web/selenium/AutocheckingRecheckDriver.java

```diff
@@ -77,17 +77,15 @@ public void get( final String url ) {
 	@Override
 	public void close() {
 		// Is this sensible? What about tests using separate sessions?
-		namingStrategy.nextTest();
-		re.cap();
```

Comment:
Hmm. I don't understand that change. Maybe ask Daniel for review, or explain to me on Monday...

Reply:
We reuse the local `cap()` method ([see here](https://github.com/retest/recheck-web/blob/a690bb0b63d635e9964921fe864305d0294ab482/src/main/java/de/retest/web/selenium/AutocheckingRecheckDriver.java#L66)), which does the same.